doc/classes/EditorScenePostImport.xml

```diff
@@ -40,7 +40,7 @@
 		    {
 		        if (node != null)
 		        {
-		            node.Name = "modified_" + node.Name;
+		            node.Name = "modified_" + string(node.Name);
```

Comment:
This is not valid C# syntax. In C# you can convert types to string using the `ToString` method but in a concatenation this should not be needed either as it's done implicitly.

Reply:
Sorry, my fault. I don't use C# and didn't test this so I've removed it